spec/lib/discourse_chat/provider/mattermost/mattermost_provider_spec.rb

```diff
@@ -17,6 +17,18 @@
       expect(stub1).to have_been_requested.once
     end
 
+    it 'uses correct logo' do
+      # Defaults to small logo url
```

Comment:
This is a description of what the test cases are testing which means that it should probably use `describe`

Reply:
As far as I can work out, `describe` is only used for 'example groups', not individual examples. How would you suggest I change this?

Reply:
```
before do
  SiteSetting.logo_small_url = "https://some_small_logo"
end

describe "when `mattermost_icon_url` is not set" do
   ... 
end

describe "when mattermost icon is set" do
  ...
end
```

Probably something like this